travis/apis.py

```diff
@@ -54,6 +56,22 @@ def __init__(self):
         self.ssh = os.environ.get(
             "WEBLATE_SSH", "ssh://user@webpage.com")
         self.tempdir = os.path.join(tempfile.gettempdir(), 'weblate_api')
+        self._ssh_keyscan()
+
+    def _ssh_keyscan(self):
+        """This method execute the command 'ssh-keysan' to avoid the
+        question when the command git clone is excecuted.
+        The question is like to:
+            'Are you sure you want to continue connecting (yes/no)?'"""
+        cmd = ['ssh-keyscan', '-p']
+        match = re.search(r'@(?P<port_host>[^/]+)', self.ssh)
+        if match:
+            port_host = match.groupdict()['port_host'].split(':')
+            host, port = ((port_host[0], 22) if len(port_host) == 1 else
+                          (port_host[0], port_host[1]))
+            cmd.extend([port, host])
+            with open(os.path.expanduser('~/.ssh/known_hosts'), 'a+') as hosts:
+                subprocess.Popen(cmd, stdout=hosts)
```

Comment:
I'm not sure how this works `stdout=host` if there is a error

@ruiztulio What do you think?

Reply:
@moylop260 
I made the follow test to check your question

![image](https://user-images.githubusercontent.com/1387970/30403305-037a77ac-98af-11e7-881d-1a97d50a7564.png)

The command `cmd = ['ssh-keyscan', '-p']`  failed because it needs more parameters

And into the `known_hosts` 
 
![image](https://user-images.githubusercontent.com/1387970/30403337-26f2fefc-98af-11e7-9fd0-cc369e83b91d.png)